applications/stream-applications-core/pom.xml

```diff
@@ -425,6 +425,10 @@
                                             <groupId>org.springframework.boot</groupId>
                                             <artifactId>spring-boot-starter-security</artifactId>
                                         </dependency>
+                                        <dependency>
+                                            <groupId>com.jayway.jsonpath</groupId>
+                                            <artifactId>json-path</artifactId>
```

Comment:
Why do we need this?
Why haven't we needed it before?
What does really make you to bring it now?

Reply:
You are right, this is not strictly related to this PR, but I found out that the apps were missing this dependency and added it here.

Reply:
OK. But there is no justification of the change in the commit message, so we could be lost bumping to the change in the future.

Reply:
Ok, will remove it from the PR.